finalPrep/modZeroHero.js

```diff
@@ -66,19 +121,63 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
 //   - energyLevel = 50 
-
+class SuperHero {
+  constructor(name, superpower, age) {
+  this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    static archNemesis = "The Syntax Error";
+    static powerLevel = 100;
+    static energyLevel = 50;
+  }
+ }
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
-
+class SuperHero {
+  constructor(name, superpower, age) {
+  this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName() {
+    console.log(this.name);
+  }
+  maximizeEnergy() {
+   console.log(this.energyLevel * 20);
+  }
+  gainPower(power) {
+   console.log(this.powerLevel * power);
```

Comment:
Nice work leveraging the function argument here.  One thing I didn't expect to see was the `console.log` here as the instructions didn't ask to log any value to the console.  I expected to see `powerLevel` updated as well, like so:

```
gainPower(power) {
    this.powerLevel += power;
}
```